### PR TITLE
awesome book with ES6

### PR DESCRIPTION
**Pull request by olanikegloria**
In this  milestone I learnt to:
Use ES6 modules to write modular JavaScript.
Learn how to use proper ES6 syntax.
Understand improvements that ES6 brings to JavaScript.
Use npm as software packages system.

### CODE REVIEW

File: modules/utilityFunction.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export function loadBookslist(bookInstanceFromBookStore) {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  function loopingBook(book) {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    button.addEventListener('click', () => {
+      const id = Number(button.getAttribute('data-book'));
+      bookInstanceFromBookStore.remove(id);
+      loadBookslist(bookInstanceFromBookStore);
+      localStorage.setItem('bookCollection', JSON.stringify(bookInstanceFromBookStore.store));
+    });
+  });
+}
+
+export function loadContacts(contactInstanceFromContact) {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  function loopingContacts(contact) {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                </div>
+              </div>`;
+  }
+
+  const contactWrapper = document.querySelector('.contact-wrapper');
+
+  contactWrapper.innerHTML = contactInstanceFromContact.profile.map((contact) => loopingContacts(contact)).join('');
+}
+
+export function addBookButton(bookInstanceFromBookStore) {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+    };
+    bookInstanceFromBookStore.add(newBook);
+    loadBookslist(bookInstanceFromBookStore);
+    titleInput.value = '';
+    authorInput.value = '';
+    localStorage.setItem('bookCollection', JSON.stringify(bookInstanceFromBookStore.store));
+  });
+}
+
+export function spaExperience() {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+          listBookSection.setAttribute('data-visible', false);
+          break;
+        default:
+          break;
+      }
+    });
+  });
+}
+
+export function loadLocalStorage(bookInstanceFromBookStore) {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

File: modules/utilityFunction.js [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+  const bookStorage = JSON.parse(localStorage.getItem('bookCollection'));
+  if (bookStorage !== null) {
+    bookInstanceFromBookStore.store = bookStorage;
+  }
+}
+
+export function loadTime() {
```
Comment (Shisui6) [inline modules/utilityFunction.js]: Please refactor this function to use arrow syntax to adhere to the project requirements🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @olanikegloria, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Nice use of modules ✅
- Good design ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by olanikegloria** [comment]
please can you show me an example because I don't know how to do it @Shisui6

File: modules/utilityFunction.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+                    <i class="fab fa-instagram-square fa-fw fa-2xl"></i>
+                  </a>
+                </div>
+              </div>`;
+  const contactWrapper = document.querySelector('.contact-wrapper');
+
+  contactWrapper.innerHTML = contactInstanceFromContact.profile.map((contact) => loopingContacts(contact)).join('');
+};
+
+export function addBookButton(bookInstanceFromBookStore) {
```
Comment (TedLivist) [inline modules/utilityFunction.js]: - [ ] You are yet to refactor this function to an arrow function. Kindly do that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @olanikegloria,

✔️ Nice UI design

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @olanikegloria, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**